gui/src/components/udx/cortx-select-create-bucket.vue

```diff
@@ -193,14 +196,11 @@ export default class CortxSelectCreateBucket extends Vue {
         createBucketForm.bucket
       );
       this.bucketUrl = res && res.data.bucket_url ? res.data.bucket_url : "NA";
-      this.s3URI = res && res.data.bucket_url ? res.data.s3_uri : "NA";
 
       this.$store.dispatch("systemConfig/hideLoader");
     }
 
-    const URI: string = this.isCreateBucket
-      ? this.s3URI
-      : this.registrationForm.bucketName.s3URI;
+    const URI: string = this.s3URI;
```

Comment:
No need of URI variable. We can directly use this.s3URI in this.emit() like so (this.$emit("onChange", false, bucketName, this.s3URI);)

Reply:
Updated the same.